modules/openy_gc_auth/modules/openy_gc_auth_daxko_sso/src/Form/TryAgainForm.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace Drupal\openy_gc_auth_daxko_sso\Form;
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpFoundation\RequestStack;
+
+/**
+ * Class TryAgainForm.
+ *
+ * @package Drupal\openy_gc_auth_daxko_sso\Form
+ */
+class TryAgainForm extends FormBase {
+
+  /**
+   * The current request.
+   *
+   * @var \Symfony\Component\HttpFoundation\Request|null
+   */
+  protected $currentRequest;
+
+  /**
+   * ConfigFactory.
+   *
+   * @var \Drupal\Core\Config\ConfigFactoryInterface
+   */
+  protected $configFactory;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct(
+    RequestStack $requestStack,
+    ConfigFactoryInterface $config_factory
+  ) {
+    $this->currentRequest = $requestStack->getCurrentRequest();
+    $this->configFactory = $config_factory;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
+    return new static(
+      $container->get('request_stack'),
+      $container->get('config.factory')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'openy_gc_auth_example_login_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $form['#prefix'] = '<h4 class="text-center">' . $this->t('You were not able to login') . '</h4>';
+
+    $form['#action'] = $this->configFactory->get('openy_gated_content.settings')->get('virtual_y_login_url');
```

Comment:
The members will only see this page if they've successfully logged in to Daxko. Redirecting to the login page from here will result in them ending up on this same page if they haven't already contacted the Y to resolve the issue that landed them here in the first place.

Reply:
I took the `#suffix` and appended it to the `#prefix` in our environment and I think that makes things make some more sense.
![VIRTUAL_Y_Login___The_West_Cook_YMCA_-__Private_Browsing_](https://user-images.githubusercontent.com/238201/100146511-a3cfaf80-2e5f-11eb-967a-3d0d9f5bcf8d.png)

Reply:
We recommend changing `You were not able to login` to `There may be a problem with your account` since technically the user is able to log in, but they're not authorized to view the content.

Reply:
@froboy that is how it looks like now.
![image](https://user-images.githubusercontent.com/3023950/100446067-d1fafc80-30b6-11eb-8e2e-6d6f1251c344.png)